fontforge/splineutil.c

```diff
@@ -2202,6 +2211,21 @@ SplinePointList *SPLCopyTransformedHintMasks(RefChar *r,
 return( _SPLCopyTransformedHintMasks(r->sc,layer,transform,basesc));
 }
 
+void SplinePointListClearCPSel(SplinePointList *spl) {
+    Spline *spline, *first;
+
+    for ( ; spl!=NULL; spl = spl->next ) {
+	first = NULL;
+	spl->first->nextcpselected = false;
+	spl->first->prevcpselected = false;
+	for ( spline = spl->first->next; spline!=NULL && spline!=first; spline=spline->to->next ) {
+	     spl->first->nextcpselected = false;
```

Comment:
For all that this iterates the spline set, isn't this always only updating the first spline point, as spl isn't updated in the inner loop?

Reply:
Yeah, I borked my copy of `SplinePointListSelect()`.